awesomeCreateApp.js

```diff
@@ -15,8 +15,12 @@ const terminalOpts = {
 }
 
 const createApp = folderName => {
-  spawn('create-react-app', [folderName], terminalOpts)
+  checkPackageInstalled('create-react-app')
+  .then(() => 
```

Comment:
Could you extract this:

```
spawn('create-react-app', [folderName], terminalOpts)
      .then( () => showQuestions(folderName))
```

create a new function ( could be called executeCreateReactApp os something related)

Wha do you think about? 😄

Reply:
Should it be an one line function as well?